emailer.py

```diff
@@ -82,6 +82,11 @@ def commit_email():
     pusher_email = '{0} <{1}>'.format(json_dict['pusher']['name'],
                                       json_dict['pusher']['email'])
 
+    commit_SHA = json_dict['after']
+    url = f"https://api.github.com/repos/chapel-lang/chapel/commits/{0}/pulls".format(commit_SHA)
+    prURL = "Unavailable"
+    prURL = requests.get(url, timeout=10).json()[0]['html_url']
```

Comment:
timeout looks good, but unless I'm missing something, I think you need to wrap in a try, because requests will throw an exception at either the connection failing, the timeout happening, the json decode failing ... 
So something like
```python
try:
    prURL = requests.get(url, timeout=10).json()[0]['html_url']
except Exception as e:
    prURL = "Unavailable"
    logger.error(f'Could not getch PR url from github: {e}')
```